website/docs/compute/local-deployment/local-deployment.md

```diff
@@ -0,0 +1,219 @@
+# Local Deployment
+
+It’s possible to deploy a Compute environment to a local environment using a local Kubernetes cluster.
+
+## Requirements
+
+- [Docker](https://www.docker.com/)
+- [Minikube](https://minikube.sigs.k8s.io/docs/start/)
+- [ngrok](https://ngrok.com/download)
+- [subo](https://github.com/suborbital/subo#installing)
+
+## Steps
+
+### 1. Create a folder for the environment
+
+This is a temporary place where we’ll create and configure our Compute environment:
+
+```bash
+mkdir my-compute; cd my-compute
```

Comment:
Any reason to have these share a line? I'd split them up.

```suggestion
mkdir my-compute
cd my-compute
```

Reply:
No reason at all, force of habit!